RunFullAnalysis.sh

```diff
@@ -247,6 +247,8 @@ if $DoHFPtSpec; then
     Channel="kDsKKpi"
   elif [ ${Particle} == "Dplus" ]; then 
     Channel="kDplusKpipi"
+  elif [ ${Particle} == "D0" ]; then
+    Channel="kD0Kpi"
   fi
```

Comment:
I think in this file you will need extra management for the raw yield extraction, where you have to pass the reflection file